tooltester/src/main/java/io/dockstore/tooltester/helper/TinyUrl.java

```diff
@@ -15,8 +15,7 @@
 
     public static String getTinyUrl(String longUrl) {
         String tinyUrlLookup = TINY_URL + longUrl;
-        try {
-            BufferedReader reader = new BufferedReader(new InputStreamReader(new URL(tinyUrlLookup).openStream(), "UTF-8"));
+        try (BufferedReader reader = new BufferedReader(new InputStreamReader(new URL(tinyUrlLookup).openStream(), "UTF-8"))) {
```

Comment:
This can probably be simplified using a dependency you already have. 
Perhaps https://commons.apache.org/proper/commons-io/description.html under IOUtils